packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -157,6 +157,69 @@ void main() {
     );
   });
 
+  testWidgets(
+      'Blur affect is disabled when disableBackgroundFilterBlur is true',
+      (WidgetTester test) async {
+    final ScrollController scrollController = ScrollController();
+    addTearDown(scrollController.dispose);
+
+    await test.pumpWidget(
+      CupertinoApp(
+        theme: const CupertinoThemeData(brightness: Brightness.light),
+        home: CupertinoPageScaffold(
+          navigationBar: const CupertinoNavigationBar(
+            middle: Text('Title'),
+            automaticBackgroundVisibility: false,
+            disableBackgroundFilterBlur: true,
+          ),
+          child: ListView(
+            controller: scrollController,
+            children: const <Widget>[
+              Placeholder(),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    expect(
+      test.widget(find.byType(BackdropFilter)),
+      isA<BackdropFilter>().having(
+          (BackdropFilter filter) => filter.enabled, 'filter enabled', false),
+    );
+  });
+
+  testWidgets(
```

Comment:
This test is a dupe of the test `Opaque background does not add blur effects, non-opaque background adds blur effects` in this file.

Reply:
You are right "Blur affect is enabled when disableBackgroundFilterBlur is false" was not needed.